### DISCUSSION TITLE
will there be a version for react-hook-form v6 supporting React 18

### DISCUSSION THREAD

**Discussion by JeomJeom** [react-hook-form/react-hook-form#8580] [RFC]
Recently, I'm trying to upgrade to React 18 on my project, but React-hook-form v6.15.8 doesn't seem to support React 18.

<img width="1038" alt="Screen Shot 2022-06-27 at 3 28 04 pm" src="https://user-images.githubusercontent.com/38584966/175866097-da7d5c40-5dc8-44f0-89bf-a9f3a8f42a5c.png">

I wonder if there will be a version for react-hook-form v6  to support React 18.

**Comment by bluebill1049** [comment]
How difficult will it be to upgrade to v7? Honestly, not sure if i will have time to update compatible with react 18 with v6 consider it's almost 2 years old.

**Reply by JeomJeom** [reply]
Thank you for prompt replying. I appreciate your taking the time to create and maintain this awesome react-hook-form. I'll consider to upgrade to v7.
